examples/aggregator/proto/aggregator.proto

```diff
@@ -0,0 +1,30 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.examples.aggregator;
+
+import "google/protobuf/empty.proto";
+
+message Vector {
+  repeated uint64 items = 1;
+}
+
+service Aggregator {
```

Comment:
Maybe it would be worth renaming this to `VectorAggregator` (and similarly for the struct), thinking that we may have other kinds of aggregator in the future. Up to you.